lib/matplotlib/widgets.py

```diff
@@ -1794,7 +1799,7 @@ def new_axes(self, ax):
         self.ax.add_patch(self.rect)
         self.artists = [self.rect]
 
-    def ignore(self, event):
+    def _ignore(self, event):
```

Comment:
You are totally right. I thought I changed where it is called as well. I will take another look.